modal-go/function.go

```diff
@@ -157,6 +175,49 @@ func processResult(ctx context.Context, result *pb.GenericResult, dataFormat pb.
 	return deserializeDataFormat(data, dataFormat)
 }
 
+// blobUpload uploads a blob to S3 and returns its ID.
```

Comment:
Should we here, and in the client, stop referencing "S3" now that it's not actually true and just say "blob storage service"?

Reply:
Yes that makes sense!